packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2052,6 +2052,19 @@ abstract class BuildContext {
   /// managing the rendering pipeline for this context.
   BuildOwner get owner;
 
+  /// Determines if [widget] is in the process of updating.
+  ///
+  /// This flag will return `true` if [widget] is in one of those life-cycles:
+  /// - [StatelessWidget.build]
+  /// - [State.build]
+  /// - [RenderObjectWidget.createRenderObject]
+  /// - [RenderObjectWidget.updateRenderObject]
+  ///
+  /// Othwerwise, it will return `false`.
+  ///
+  /// Always Returns `null` in release mode.
```

Comment:
From the implementation it looks as if it will always return `false` in release mode?

Reply:
Oh, yes, I changed the code a bit. Will fix 👌